src/scripts/load_python_libs.sh

```diff
@@ -3,3 +3,19 @@
 ./load_openssl.sh
 cd /home/lind/lind_project/tests/applications/python
 ./fastcompile.sh
+
+# Copies 
+echo Copying files...
+libsarray=('_collections' 'operator' 'itertools' '_struct' 'math' 'binascii' 'time' 'cStringIO' '_random' 'array' '_socket' '_functools' 'cPickle' '_struct' 'select' 'array' 'unicodedata')
+length=${#libsarray[@]}
+for (( i = 0; i < length; i++ ));
```

Comment:
Again you should iterate over elements not indices

Reply:
Done.